.eslintrc.json

```diff
@@ -1,8 +1,9 @@
 {
-  "extends": ["react-app", "plugin:prettier/recommended"],
+  "extends": ["eslint:recommended", "react-app"],
+  "plugins": ["eslint-plugin-simple-import-sort", "prettier"],
   "rules": {
-    "no-unused-vars": ["error"],
```

Comment:
I removed it because it is a default rule (see [rules](https://eslint.org/docs/rules/) and [no-unused-vars](https://eslint.org/docs/rules/no-unused-vars))

Reply:
ah cool. I figured as much. thanks for sharing